docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,140 @@
+# Scalar DL Backup Creation and Restoration
+
+Scalar DL backup should be created after the Scalar DL server is paused otherwise there is a chance for inconsistency. 
+This guide shows you how to create and restore Scalar DL data backup without any data inconsistency.
+
+## Prerequisites
+
+* Read the Guide on How to [Back up and Restore Databases Integrated with Scalar DB](https://github.com/scalar-labs/scalardb/blob/master/docs/backup-restore.md)
```

Comment:
The hypertext is wrong. (should include `Guide on How to`)

Reply:
Thank you for your suggestion.
Suggestion integrated cd32752